templates/primary-secondary/primary-secondary.js

```diff
@@ -17,19 +18,33 @@ class TemplatePrimarySecondary extends LitElement {
 			:host([hidden]) {
 				display: none;
 			}
+			:host([width-type="normal"]) .content-container,
+			:host([width-type="normal"]) .footer-container {
+				max-width: 1230px;
+				margin: 0 auto;
+			}
 			.container {
 				display: grid;
-				grid-template-columns: minmax(320px, 2fr) 1px minmax(320px, 1fr);
+				grid-template-columns: auto;
 				grid-template-rows: auto 1fr auto;
 				grid-template-areas:
-				"header header header"
-				"primary divider secondary"
-				"footer footer footer";
+				"header"
+				"content"
+				"footer";
 				height: 100vh;
 			}
 			header {
 				grid-area: header;
 			}
+			.content-container {
```

Comment:
Can you rename this CSS class to `d2l-template-primary-secondary-content-container`? I know it's a mouthful, but we like to scope our CSS classes for browsers who don't yet support shadow DOM -- otherwise someone else could define styles for a `content-container` CSS class and they'd leak in (or vice versa).

Same for `footer-container`.